libdedx/dedx.c

```diff
@@ -417,8 +416,29 @@ int _dedx_check_ion(int prog, int ion) {
 	return 0;
 }
 
+float set_conversion_rate(const int old_unit, const int new_unit, const int material, int *err) {
```

Comment:
This function doesn't set any state on any object, why not call it simply `conversion_factor` ?
I do not like `rate` as it resembles some process which happens at some pace, `factor` is more commonly used